src/main/java/org/osc/sdk/manager/api/ManagerSecurityGroupInterfaceApi.java

```diff
@@ -34,26 +35,21 @@
     /**
      * Creates a security group interface within context of the current virtual system
      *
-     * @param name  the name of the security group interface
-     * @param policyId  the identifier of the policy
-     * @param tag  the tag associated with the policy defined by OSC
-     * @return the identifier of the created policy
+     * @param sgiElem  provides context information of the SecurityGroupInterface
```

Comment:
the information needed to create a security group interface.
Note:  Avoid using type names on the javadoc unless it is a link or a see. For other cases we should be cursive, i.e.: security group interface